tests/integration/ceph_mgr_test.go

```diff
@@ -78,51 +78,74 @@ func (suite *MgrSmokeSuite) SetupSuite() {
 	suite.namespace = "mgr-ns"
 
 	mgrTestCluster := TestCluster{
-		namespace:        suite.namespace,
-		storeType:        "bluestore",
-		storageClassName: "",
-		useHelm:          false,
-		usePVC:           false,
-		mons:             3,
-		rbdMirrorWorkers: 1,
-		rookCephCleanup:  true,
-		withoutOSDs:      true,
+		namespace:               suite.namespace,
+		storeType:               "bluestore",
+		storageClassName:        "",
+		useHelm:                 false,
+		usePVC:                  false,
+		mons:                    1,
+		rbdMirrorWorkers:        0,
+		rookCephCleanup:         true,
+		skipOSDCreation:         true,
+		minimalMatrixK8sVersion: smokeSuiteMinimalTestVersion,
+		rookVersion:             installer.VersionMaster,
+		cephVersion:             installer.MasterVersion,
 	}
 
-	suite.op, suite.k8sh = StartTestCluster(suite.T, smokeSuiteMinimalTestVersion, &mgrTestCluster, installer.VersionMaster, installer.MasterVersion)
-
-	suite.CLI = MgrCLI{k8sh: suite.k8sh, namespace: suite.namespace}
-
+	suite.cluster, suite.k8sh = StartTestCluster(suite.T, &mgrTestCluster)
+	suite.waitOrchestrator()
```

Comment:
Are you seeing a need to wait? Since `StartTestCluster()` creates the cluster and the toolbox I would think everything would be ready, although maybe the toolbox needed a few more seconds to initialize?

Reply:
Yes... i was debugging this a couple of hours... despite the toolbox container is running the rook manager module is not available since the firs moment.  It takes a few seconds to have it operative... I checked that for example you can run intermediately a "ceph status" command without problems, but any "ceph orch ..."  command fails with "error 22"  during  a few seconds after the container is running.